### PR TITLE
[blog] post for gatsby-plugin-page-creator

### PR DESCRIPTION
**Pull request by nodox**
New blog post for [gatsby-plugin-page-creator](https://www.gatsbyjs.org/packages/gatsby-plugin-page-creator/?=page-creator).

### CODE REVIEW

File: docs/blog/2018-07-07-the-gatsby-plugin-page-creator/index.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+![Gatsby plugin page creator in the plugin library](plugin-library-page-creator.png)
+
+In this article we will talk about a new plugin that let’s you auto create
+pages from components in any folder within your Gatsby project! We will chat about the
+problem that sparked the creation of this plugin. Afterwards, we'll talk about how you can
+use this plugin to auto create pages without having to place your components in the **src/pages** directory.
+
+# The problem with static values
+Here's how it all got started. The goal with [Gatsby Manor](https://gatsbymanor.com)
```
Comment (KyleAMathews) [inline docs/blog/2018-07-07-the-gatsby-plugin-page-creator/index.md]: Could you make it clear this is a separate project you run? Sometimes people think Gatsby Manor is an official project so we should make it clear it's not.

File: docs/blog/2018-07-07-the-gatsby-plugin-page-creator/index.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+One [issue](https://github.com/gatsbyjs/gatsby/issues/2514) raised how
+uncommon this pattern might be when handing a project over to other
+client that are less technical. Each issue focused on a valid concern, with
+many community members in support of a better convention. With this data,
+I set out to determine a solution for this problem.
+
+# The great pull request
+![Pull request for gatsby plugin page creator](page-creator-pull-request.png)
+
+With the help of Kyle Matthews, he directed me to the location of the
```
Comment (KyleAMathews) [inline docs/blog/2018-07-07-the-gatsby-plugin-page-creator/index.md]: *Mathews ;-)

File: docs/blog/2018-07-07-the-gatsby-plugin-page-creator/index.md (hunk 1 above)
Comment (nodox) [reply docs/blog/2018-07-07-the-gatsby-plugin-page-creator/index.md]: For sure! Thanks for letting me grow along side the community. I think the more starters we make the more people can jump onto the Gatsby wagon!

**Comment by m-allanson** [comment]
Preview link: https://deploy-preview-6339--next-gatsbyjs.netlify.com/blog/2018-07-07-the-gatsby-plugin-page-creator/

**Comment by nodox** [comment]
@m-allanson Those edits LGTM! 🚢. Thanks for fixing the dead link!

**Comment by nodox** [comment]
@KyleAMathews  @m-allanson ready to go?

**Comment by m-allanson** [comment]
Yep, apologies for the delay. Merging now!

**Comment by m-allanson** [comment]
Yep, apologies for the delay. Merging now!

**Comment by m-allanson** [comment]
Cherry-picked to `v1`

**Pull request merged**